jbmc/unit/java_bytecode/java_bytecode_parser/parse_class_without_instructions.cpp

```diff
@@ -0,0 +1,69 @@
+/*******************************************************************\
+
+ Module: Unit tests to parse a class without its methods' instructions
+
+ Author: Diffblue Ltd.
+
+\*******************************************************************/
+
+#include <java-testing-utils/load_java_class.h>
+#include <java_bytecode/java_bytecode_parse_tree.h>
+#include <java_bytecode/java_bytecode_parser.h>
+#include <util/message.h>
+
+#include <testing-utils/catch.hpp>
+
+SCENARIO(
+  "java_bytecode_parse_class_without_instructions",
+  "[core][java_bytecode][java_bytecode_parser]")
+{
+  null_message_handlert null_out;
+
+  WHEN("Loading a class without instructions")
+  {
+    auto loaded =
+      java_bytecode_parse(
+        "./java_bytecode/java_bytecode_parser/Trivial.class", null_out, true);
+    THEN("Loading should succeed")
+    {
+      REQUIRE(loaded);
+      const auto &loaded_class = *loaded;
+
+      THEN("It should have one method")
+      {
+        REQUIRE(loaded_class.parsed_class.methods.size() == 1);
+
+        const auto &method = *loaded_class.parsed_class.methods.begin();
```

Comment:
:bulb: Add a check here that the loaded class at least has the member variable `x`
I would also extend to check:
 - [ ] accessibility of class
 - [ ] inner class loaded
 - [ ] accessibility of method
 - [ ] annotations on method
 - [ ] accessibility / annotations of fields
 - [ ] type hierarchy 
 - [ ] a non constructor